sdk/compute/Microsoft.Azure.Management.Compute/tests/ScenarioTests/GalleryTests.cs

```diff
@@ -415,6 +417,60 @@ public void Gallery_WithSharingProfile_CRUD_Tests()
             }
         }
 
+        [Fact]
+        public void Gallery_SharingToCommunity_CRUD_Tests()
+        {
```

Comment:
please update this test and Gallery_SharingToSubscriptionAndTenant_CRUD_Tests to have a finally block at the end to delete the resourcegroup, whether the test passes or not. Please see: https://github.com/Azure/azure-sdk-for-net/blob/dcfa27a8b6fc9d5e15db7a85d5a656616bdf97fe/sdk/compute/Microsoft.Azure.Management.Compute/tests/ScenarioTests/OSProfileTests.cs#L286

Reply:
added in case bad things happen, thanks!